core/arch/arm/plat-stm32mp2/stm32_util.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2022, STMicroelectronics
+ */
+
+#ifndef __STM32_UTIL_H__
+#define __STM32_UTIL_H__
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <types_ext.h>
```

Comment:
`#include <kernel/spinlock.>`
and remove inclusions of _assert.h_, _stdbool.h_

Reply:
Ok but I'll add a patch on top of this series to add missing assert.h include in stm32_opp driver.

Seen by trying to remove 
#include <assert.h>
#include <stdbool.h>

and not adding:
#include <kernel/spinlock.h>

As assert.h is included in #include <kernel/spinlock.h>, it's not seen otherwise.